src/controllers/admin/webhook.ts

```diff
@@ -43,69 +57,356 @@ export class WebhookController {
 			} satisfies INotifyMessage);
 			return response.sendStatus(StatusCodes.BAD_REQUEST);
 		}
-		// Handle the event
-		switch (event.type) {
-			case 'customer.subscription.trial_will_end':
-				subscription = event.data.object;
-				status = subscription.status;
-				await eventTracker.notify({
-					message: EventTracker.compileBasicNotification(
-						`Subscription status is ${status} for subscription with id: ${subscription.id}`,
-						'Stripe Webhook: customer.subscription.trial_will_end'
-					),
-					severity: 'info',
-				} satisfies INotifyMessage);
-				await eventTracker.submit(
-					buildSubmitOperation(subscription, OperationNameEnum.SUBSCRIPTION_TRIAL_WILL_END)
+
+		try {
+			// Handle the event
+			switch (event.type) {
+				case 'customer.subscription.trial_will_end':
+					subscription = event.data.object;
+					status = subscription.status;
+					await eventTracker.notify({
+						message: EventTracker.compileBasicNotification(
+							`Subscription status is ${status} for subscription with id: ${subscription.id}`,
+							'Stripe Webhook: customer.subscription.trial_will_end'
+						),
+						severity: 'info',
+					} satisfies INotifyMessage);
```

Comment:
@DaevMithran should we not perform some action on this one? if not, then might as well remove this?

Reply:
Removed